components/event-feed-service/pkg/integration_test/project_update_test.go

```diff
@@ -0,0 +1,414 @@
+package integration_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/chef/automate/api/interservice/authz"
+	"github.com/chef/automate/components/event-feed-service/pkg/feed"
+	"github.com/chef/automate/components/event-feed-service/pkg/persistence"
+	event "github.com/chef/automate/components/event-service/config"
+)
+
+func TestProjectUpdatePainlessElasticsearchScript(t *testing.T) {
```

Comment:
Most of these test came from here https://github.com/chef/automate/blob/4d90b019189761ea8fe0e833a4b0b95ff862c3df/components/ingest-service/integration_test/project_update_actions_test.go#L16